conf/munin.config

```
@@ -20,6 +20,13 @@ process {
   maxForks = 46
 }
 
+process {
+  withName:Mutect2 {
+    time = { check_max( 48.h * task.attempt, 'time' ) }
```

```suggestion
    time = {48.h * task.attempt}
```
I don't think `check_max` will work there